### PR TITLE
android: switch back to NativeActivity.

### PR DESCRIPTION
**Pull request by Keenuts**
NativeActivity is supported since a long time. GameActivity is the new recommended method.
THe issue is GameActivity is not supported by Android, and being brand new means lack of support in other domains.

Moving back to NativeActivity is a risk, if it suddenly gets deprecated. But the cost of the switch being lower than the cost of fixing dependencies to use GameActivity means it's an acceptable risk.

### CODE REVIEW

**Comment by Keenuts** [comment]
@at-ninja : are you OK with this PR? (cannot add you as reviewer)

File: CMakeLists.txt [hunk 1]
```diff
@@ -198,7 +198,11 @@ endif()
 # Add game-activity for ANDROID
```
Comment (apazylbe) [inline CMakeLists.txt:198]: Should we update this comment?

**Comment by apazylbe** [comment]
I think @hysw had a similar change, so adding in case he has any comments

**Review by hysw — APPROVED** [review]
This is essentially the same change I had

**Review by at-ninja — COMMENTED** [review]
LGTM, thanks for the heads up.

**Pull request merged**